lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -581,6 +583,7 @@ sub job_grab {
 
     # TODO: cleanup previous tmpdir
     $worker->set_property('WORKER_TMPDIR', tempdir());
+    OpenQA::IPC->ipc->emit_signal('scheduler', 'job_grab', $job_hashref);
```

Comment:
Yes, but I was thinking we can limit access to sensitive signals in dbus policy file.
My idea was to add as much info to signals as I can to limit round trips. (get id -> ask details).

But in this case, one probably does not need info how to connect to WebUI so I'll mask it.